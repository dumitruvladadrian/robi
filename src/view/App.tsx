import React from 'react';

export const App = () => <p>Hi there !</p>;