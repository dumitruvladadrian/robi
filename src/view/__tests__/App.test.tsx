import '@testing-library/jest-dom/extend-expect';
import {render} from '@testing-library/react';
import React from 'react';
import {App} from '../App';

test('Checking App snapshot', () => {
	const app = render(
		<App/>
	);
	expect(app.container.firstChild).toMatchSnapshot();
});
