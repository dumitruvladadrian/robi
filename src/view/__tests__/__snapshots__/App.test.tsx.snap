// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking App snapshot 1`] = `
<p>
  Hi there !
</p>
`;
